src/core/Song.cpp

```diff
@@ -69,6 +69,7 @@ Song::Song() :
 	m_oldTicksPerBar( DefaultTicksPerBar ),
 	m_masterVolumeModel( 100, 0, 200, this, tr( "Master volume" ) ),
 	m_masterPitchModel( 0, -12, 12, this, tr( "Master pitch" ) ),
+	m_nLoadingTrack( 0 ),
```

Comment:
The changes in `Song.cpp` look sane, I just wonder, is this related to the valgrind output in #5409?

Reply:
Yes. I executed valgrind with  --track-origins=yes. It pointed a line that has new Song, expression which performs default initialization.